docs/docker_container.md

```diff
@@ -485,10 +485,11 @@ Below is a config example using heterogeneous plugin with GPU as a primary devic
 
 ## Security Considerations <a name="sec"></a>
 
-OpenVINO Model Server docker containers, by default, starts with the security context of local account ovms with linux uid 5000. It ensure docker container has not elevated permissions on the host machine. This is in line with best practices to use minimal permissions to run docker applications. You can change the security context by adding --user parameter to docker run command. It might be needed for example to load mounted models with restricted access. For example:
+OpenVINO Model Server docker containers, by default, starts with the security context of local account ovms with linux uid 5000. It ensure docker container has not elevated permissions on the host machine. This is in line with best practices to use minimal permissions to run docker applications. You can change the security context by adding --user parameter to docker run command. It might be needed for example to load mounted models with restricted access. 
+For additional security hardening, you might also consider preventing write operations on the container root filesystem by adding a --read-only flag. It might prevent undesired modification of the container files. It case the cloud storage is used for the models (S3, GoogleStorage or Azure storage), restricting root filesystem should be combined with  `--tmpfs /tmp` flag.
 
 ```
-docker run --rm -d  --user $(id -u):$(id -g)  -v ${pwd}/model/:/model -p 9178:9178 openvino/model_server:latest \
+docker run --rm -d  --user $(id -u):$(id -g) --read-only --tmpfs /tmp -v ${pwd}/model/:/model -p 9178:9178 openvino/model_server:latest \
```

Comment:
There are more how-to's with docker run command in our documentation directory. Do we need to add it to every single doc?